test/org/opensolaris/opengrok/web/UtilTest.java

```diff
@@ -343,12 +358,19 @@ public void testLinkify() {
         assertTrue(Util.linkify("smtp://example.com/OpenGrok/OpenGrok").equals("smtp://example.com/OpenGrok/OpenGrok"));
         assertTrue(Util.linkify("just some crazy text").equals("just some crazy text"));
 
-        // escaping tests
+        // escaping url
+        assertTrue(Util.linkify("http://www.example.com/\"quotation\"/else")
+                .contains("href=\"" + Util.encodeURL("http://www.example.com/\"quotation\"/else") + "\""));
+        assertTrue(Util.linkify("https://example.com/><\"")
+                .contains("href=\"" + Util.encodeURL("https://example.com/><\"") + "\""));
+        assertTrue(Util.linkify("http://www.example.com?param=1&param2=2&param3=\"quoted>\"")
+                .contains("href=\"" + Util.encodeURL("http://www.example.com?param=1&param2=2&param3=\"quoted>\"") + "\""));
+        // escaping titles
         assertTrue(Util.linkify("http://www.example.com/\"quotation\"/else")
-                .contains("href=\"http://www.example.com/%22quotation%22/else\""));
+                .matches(".*title=\".*?" + Util.encode("http://www.example.com/\"quotation\"/else") + ".*\".*"));
```

Comment:
I added `contains` as it is with the `href` attribute with the exact content "Link to 'url'". I don't think it's the best solution to have it like the whole content compassion as explained above.